l10n_it_declaration_of_intent/models/account_move.py

```diff
@@ -106,11 +109,15 @@ def _post(self, soft=True):
                 continue
             # Group lines by tax
             grouped_lines = self.get_move_lines_by_declaration(lines)
-            invoice.update_declarations(declarations, grouped_lines)
+            invoice.update_declarations(
+                declarations, grouped_lines, declarations_used_amounts
+            )
 
         return posted
 
-    def update_declarations(self, declarations, grouped_lines):
+    def update_declarations(
+        self, declarations, grouped_lines, declarations_used_amounts
+    ):
```

Comment:
```suggestion
    def update_declarations(
        self, declarations_used_amounts, grouped_lines
    ):
```
`declarations` non è più usato dal metodo e comunque le sue informazioni sono recuperabili da `declarations_used_amounts`